Desription: docbook2x-man does not create output that lintian is delighted by
Author: Thorsten Alteholz <debian@alteholz.de>

Index: ent/Makefile
===================================================================
--- ent.orig/Makefile	2012-06-05 13:30:52.000000000 +0200
+++ ent/Makefile	2012-06-05 13:32:42.000000000 +0200
@@ -4,7 +4,7 @@
 all: ent ent.1
 
 ent: ent.o iso8859.o randtest.o
-	$(CC) $(CFLAGS) ent.o iso8859.o randtest.o -o ent -lm
+	$(CC) $(CFLAGS) $(LDFLAGS) ent.o iso8859.o randtest.o -o ent -lm
 
 ent.c: iso8859.h randtest.h
 
@@ -28,4 +28,11 @@
 %.1: %.1.txt
 	asciidoc -b docbook -d manpage $<
 	docbook2x-man $*.1.xml
-	rm -f $*.1.xml
+	mv $*.1 $*.1.pre
+	cat $*.1.pre | sed "s/-b /\\\\-b/" | \
+		sed "s/-c /\\\\-c/" | \
+		sed "s/-f /\\\\-f/" | \
+		sed "s/-t /\\\\-t/" | \
+		sed "s/-u /\\\\-u/" | \
+		sed "s/ -b/ \\\\-b/" > $*.1
+	rm -f $*.1.xml $*.1.pre
